cmd/arktos-network-controller/app/controller_test.go

```diff
@@ -197,7 +194,8 @@ func TestManageNonFlatNetwork(t *testing.T) {
 				},
 				Status: corev1.ServiceStatus{},
 			},
-			expectingError: false,
+			expectingError:   false,
+			expectedNetPhase: "Pending",
```

Comment:
v1.NetworkingPending?

Please also check the hard-coded net phases and replace them with the const.

Reply:
test code is fine to have string literal.